Dockerfile

```diff
@@ -4,11 +4,21 @@ LABEL authors="mishok2503"
 RUN apt-get update && apt-get -y --no-install-recommends install \
     build-essential \
     cmake \
-    make
+    make \
+    git \
+    ca-certificates
+
+RUN git config --global http.sslVerify false
```

Comment:
Could you please explain what this line and one above are for?

Reply:
I deleted "RUN git config --global http.sslVerify false" because this command is redundant. But "ca-certificates" is needed. You receive "Failed to clone repository: 'https://github.com/catchorg/Catch2.git'" error message without this command